rpc/wrtc_peer.go

```diff
@@ -53,6 +54,17 @@ func newWebRTCAPI(isClient bool, logger golog.Logger) (*webrtc.API, error) {
 	// while the client (controlling) provides an mDNS candidate that may resolve to 127.0.0.1.
 	settingEngine.SetIncludeLoopbackCandidate(true)
 	settingEngine.SetRelayAcceptanceMinWait(3 * time.Second)
+	settingEngine.SetIPFilter(func(ip net.IP) bool {
+		// RSDK-239: Disallow ipv6 connections. Viam <-> GRPC has a compatibility
```

Comment:
don't think we need the ticket number here anymore, and maybe copying this comment (https://github.com/viamrobotics/goutils/blob/2fc19c30aeea5218c16df7255999da8a1bc145c7/rpc/dial.go#L245) would be more useful that just saying compatibility issue?

Reply:
What is "IPv6 with scope"?

Reply:
not super sure (5minutes of googling indicate it is related to reachability https://superuser.com/a/99753), seems like eric was referring to https://github.com/grpc/grpc-go/issues/3272 tho, which may be fixed soon ™️ ?

we should link to the grpc-go github issue for the context it provides

Reply:
Ah nice; linked to `grpc-go` issue and took out the ticket reference.

Reply:
Cool, I'm good with removing the ticket number for a reference to the grpc issue link.